eng/Versions.props

```diff
@@ -85,7 +85,6 @@
     <!-- Dependencies from https://github.com/dotnet/runtime -->
     <SystemCodeDomPackageVersion>7.0.0-preview.2.22116.9</SystemCodeDomPackageVersion>
     <SystemTextEncodingCodePagesPackageVersion>7.0.0-preview.2.22116.9</SystemTextEncodingCodePagesPackageVersion>
-    <SystemSecurityCryptographyProtectedDataPackageVersion>5.0.0-preview.7.20364.11</SystemSecurityCryptographyProtectedDataPackageVersion>
```

Comment:
Shouldn't we just update this version. Presumably this property is used somewhere else in the code, or not all?

Reply:
We're already doing that in the first version block at the top of this file - this property came from the runtime ages ago, and from what I could tell in their versions.props that don't even have this dependency anymore.

This being set was overriding our top-level 6.0 stable dependency.

Reply:
(See Line 33 of this same file)